pkg/api/2018-09-30-preview/api/types.go

```diff
@@ -8,13 +12,15 @@ const (
 // OpenShiftManagedCluster complies with the ARM model of resource definition in
 // a JSON template.
 type OpenShiftManagedCluster struct {
+	autorest.Response `json:"-"`
+
```

Comment:
should we be changing this? I thought that these versioned types were locked in stone.

Reply:
The only change is the addition of autorest.Response that is included in the SDK type and we were missing. The rest of the diff matches the order of our fields with that of the SDK type.